src/io/flutter/settings/FlutterSettings.java

```diff
@@ -192,6 +193,16 @@ public void setVerboseLogging(boolean value) {
     fireEvent();
   }
 
+  public boolean isMemoryProfiler() {
```

Comment:
isMemoryProfiler sounds a bit odd as FlutterSettings is not a memory profiler. Perhaps useMemoryProfiler or
something long like
isAndroidStudioMemoryProfilerSupported()
?

Reply:
or
isMemoryProfilerEnabled